ocs_ci/utility/utils.py

```diff
@@ -408,6 +410,21 @@ def run_cmd(cmd, secrets=None, **kwargs):
     return mask_secrets(r.stdout.decode(), secrets)
 
 
+def run_mcg_cmd(cmd, namespace='openshift-storage'):
```

Comment:
namespace=None

and in the function please use:

namespace = nameaspace if namespace else config.ENV_DATA['cluster_namespace']